.circleci/config.yml

```diff
@@ -48,10 +48,18 @@ jobs:
     docker:
       - image: node:9
     <<: *unit_tests
+  node10:
```

Comment:
Should we just remove `node4 and `node9`?

Reply:
That would make this semver-major, and I didn't feel it is costing us anything to keep circle ci building those =D.

Reply:
SGTM